AndroidXCI/build.gradle.kts

```diff
@@ -22,14 +21,20 @@ plugins {
     id("org.jlleitschuh.gradle.ktlint-idea") version "10.0.0"
 }
 
-group = "dev.androidx.build.ci"
-version = "1.0-SNAPSHOT"
-
 repositories {
     mavenCentral()
     google()
 }
 
+val githubRunId = project.providers.environmentVariable("GITHUB_RUN_ID")
+val publishVersion = githubRunId.orElse("").map {
+    if (it == "") {
+        "1.0-SNAPSHOT"
```

Comment:
should we use regular versioning of X.Y.Z ?

Reply:
talked offline, we'll keep it as is for now.